src/shared/stores/unsaved-data-store.js

```diff
@@ -0,0 +1,36 @@
+import create from 'zustand'
+import { persist } from 'zustand/middleware'
+
+const inititalState = {
+    unsavedComments: {},
+    unsavedLimits: {},
```

Comment:
Minor and just my opinion, but I would argue that `unsaved`  is implied by the name of the store. So `comments` and `limits` would suffice.